index.bs

```diff
@@ -1080,6 +1088,15 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 		</div><!-- algorithm -->
 
+<h2 id="feature-policy-integration">Feature Policy Integration</h2>
+
+This specification defines two [=policy-controlled feature=]s identified by the
```

Comment:
```suggestion
This specification defines two [=policy-controlled feature|policy-controlled features=] identified by the
```

Reply:
A minor thing... if you haven't already, try just [=policy-controlled features=] ... BS is pretty smart about matching plurals, so it might just work.

Reply:
Thanks! [=policy-controlled-features=] did work.